ipn/ipnlocal/web_client.go

```diff
@@ -88,3 +94,38 @@ func (b *LocalBackend) handleWebClientConn(c net.Conn) error {
 	s := http.Server{Handler: b.webClient.server}
 	return s.Serve(netutil.NewOneConnListener(c, nil))
 }
+
+func (b *LocalBackend) updateWebClientTCPPortNetMapAddrListenersLocked() {
```

Comment:
This function makes sense from reading it, but the name is so long 😅 maybe add a little doc on this